src/core/device-interface.h

```diff
@@ -70,6 +70,8 @@ class device_interface
 
     virtual bool contradicts( const stream_profile_interface * a,
                               const std::vector< stream_profile > & others ) const = 0;
+
```

Comment:
@OhadMeir  any idea how this added an API w/o adding it to the include headers?

Reply:
realsense-viewer.cpp added an include to `api.h` which, despite the name, is actually under `src` folder. This bypass the API mechanism.